.github/workflows/reuse.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
             ${{ runner.os }}-go-
 
       - name: Log into ghcr.io
-        uses: docker/login-action@master
+        uses: docker/login-action@7c79b598eaa33458e78e8d0d71e0a9c217dd92af
```

Comment:
This also works but I would have suggested using the commit of [stable release](https://github.com/docker/login-action/releases/tag/v1.14.1) instead of master.